lighthouse-core/audits/dobetterweb/doctype.js

```diff
@@ -15,8 +15,8 @@ const UIStrings = {
   failureTitle: 'Page lacks the HTML doctype, thus triggering quirks-mode',
   /** Description of a Lighthouse audit that tells the user why they should define an HTML doctype. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Specifying a doctype prevents the browser ' +
-    'from switching to quirks-mode. Read more on the ' +
-    '[MDN Web Docs page](https://developer.mozilla.org/en-US/docs/Glossary/Doctype)',
```

Comment:
man we're really not making good progress on getting rid of `Learn more` are we team 😆 

(I think this is the right thing to do here @mfriesenhahn, just a bummer that we've been talking about following our own advice for a long time and it's sad to see one more convert to "Learn more")